src/registry.json

```diff
@@ -1960,9 +1963,9 @@
         "name": "Aleph Zero Wallet",
         "author": {
           "name": "Bide",
-          "website": "https://bide.dev"
+          "website": "https://azero.dev/"
```

Comment:
This change does not make sense. Is the developer Bide? If so, this should be Bide's website.

Reply:
Done.